packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1010,7 +1035,7 @@ export default class SalesforceCRMService implements CRM {
 
       const user = userQuery.records[0] as { Email: string };
 
-      return user.Email;
+      return { email: user.Email, recordType: RoutingReasons.ACCOUNT_LOOKUP_FIELD };
```

Comment:
This method is the lookup field under an account. Since this is different from account ownership it should have it's own reason.